agcaballero/portfolio/src/main/webapp/comments.html

```diff
@@ -51,14 +53,120 @@ <h1>Leave a Comment</h1>
+              <option value="KO">Korean</option>
+              <option value="KU">Kurdish</option>
+              <option value="KY">Kyrgyz</option>
+              <option value="LO">Lao</option>
+              <option value="LA">Latin</option>
+              <option value="LV">Latvian</option>
+              <option value="LT">Lithuanian</option>
+              <option value="LB">Luxembourgish</option>
+              <option value="MK">Macedonian</option>
+              <option value="MS">Malay</option>
+              <option value="ML">Malayalam</option>
+              <option value="MT">Maltese</option>
+              <option value="MI">Maori</option>
+              <option value="MR">Marathi</option>
+              <option value="MN">Mongolian</option>
+              <option value="MY">Myanmar (Burmese)</option>
+              <option value="NE">Nepali</option>
+              <option value="NO">Norwegian</option>
+              <option value="NY">Nyanja (Chichewa)</option>
+              <option value="OR">Odia (Oriya)</option>
+              <option value="PS">Pashto</option>
+              <option value="FA">Persian</option>
+              <option value="PL">Polish</option>
+              <option value="PT">Portuguese</option>
+              <option value="PA">Punjabi</option>
+              <option value="RO">Romanian</option>
+              <option value="RU">Russian</option>
+              <option value="SM">Samoan</option>
+              <option value="GD">Scots Gaelic</option>
+              <option value="SR">Serbian</option>
+              <option value="ST">Sesotho</option>
+              <option value="SN">Shona</option>
+              <option value="SD">Sindhi</option>
+              <option value="SI">Sinhala (Sinhalese)</option>
+              <option value="SK">Slovak</option>
+              <option value="SL">Slovenian</option>
+              <option value="SO">Somali</option>
+              <option value="ES">Spanish</option>
+              <option value="SU">Sundanese</option>
+              <option value="SW">Swahili</option>
+              <option value="SV">Swedish </option>
+              <option value="TL">Tagalog (Filipino)</option>
+              <option value="TG">Tajik</option>
+              <option value="TA">Tamil</option>
+              <option value="TT">Tatar</option>
+              <option value="TE">Telugu</option>
+              <option value="TH">Thai</option>
+              <option value="TR">Turkish</option>
+              <option value="TK">Turkmen</option>
+              <option value="UK">Ukrainian</option>
+              <option value="UR">Urdu</option>
+              <option value="UZ">Uzbek</option>
+              <option value="VI">Vietnamese</option>
+              <option value="CY">Welsh</option>
+              <option value="XH">Xhosa</option>
+              <option value="YI">Yiddish</option>
+              <option value="YO">Yoraba</option>
+              <option value="ZU">Zulu</option>
```

Comment:
Consider starting off with just a few languages that you're able to test for accuracy.

Reply:
I already tested a couple languages for accuracy, which is why I went ahead to add all the languages supported by the Translate API. @ykjchen are there other tests you suggest I run to ensure that this program works?

Reply:
In general, it may be a larger maintenance cost to support more languages (users may write in about translations you're not familiar with), but as long as you've taken this into account and are okay with the tradeoff, I think this is fine. Feel free to resolve this comment after reading.

Reply:
Thanks. That clarification makes sense. I just wanted to try and be accessible, so I think I'll keep the expanded languages. However. I appreciate understanding the trade off better.